contracts/child/ChildChainManager/ChildChainManager.sol

```diff
@@ -76,15 +81,18 @@ contract ChildChainManager is IChildChainManager, Initializable, AccessControl {
     }
 
     function _syncDeposit(bytes memory syncData) private {
-        (address user, address rootToken, bytes memory depositData) = abi
-            .decode(syncData, (address, address, bytes));
+        (address user, address rootToken, bytes memory depositData, address callback) = abi
```

Comment:
Should we move `callback` before `depositData`?

Reply:
It doesn't matter to me.